package-lock.json

```diff
@@ -13384,13 +13384,18 @@
                 "node": ">=14"
             }
         },
+        "packages/node/node_modules/@backtrace-labs/sdk-core": {
```

Comment:
I think you didn't bump everything

Reply:
Or did you mean, bump the version but skip the actual node release for now?

Reply:
I went ahead and included node

Reply:
Yeah this is exactly what I meant. Do not release a new node library yet